### DISCUSSION TITLE
正确的 QoS 解决方案

### DISCUSSION THREAD

**Discussion by Kaizz983** [sbwml/r4s_build_script#79] [Ideas]
我看到你集成了 natflow 包：
# Natflow
CONFIG_PACKAGE_natflow=m
你可以考虑集成 X-WRT 的“简单 QoS”或类似解决方案，它可以限制整个范围内每个用户 IP 的速度，而无需手动设置。另一方面，基于 natflow 的 X-WRT 的 QoS 解决方案也消耗更少的 CPU

**Comment by sbwml** [comment]
做不了，我对 Qos 这个东西非常陌生。甚至不清楚它的作用是实现些什么东西。
